libs/npm/core/src/components/grid-table/Pager.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { BiFirstPage, BiLastPage } from 'react-icons/bi';
 
-import { Button, ButtonVariant } from '../button';
-import { Row } from '../flex';
-import { Text } from '../form/text';
-import { Show } from '../show';
+import { Button, ButtonVariant } from 'tno-core';
```

Comment:
Cannot self reference

Reply:
Those changes weren't supposed to be there. The commit now shows how I wanted it to look.